ts/Stock/StockToolsGui.ts

```diff
@@ -311,6 +311,8 @@ setOptions({
                 background: 'Background',
 
                 // Indicators' params (#15170):
+                'search-indicators': 'Search Indicators',
```

Comment:
`searchIndicators`*

Reply:
That is the tricky part because for now the `search-indicators` is used also as a class for that input.
But I guess that I might implement a separate mechanism to add the class name.

Reply:
Or, change the name. For example `indicatorsSearchBox`, or something similar 🤔